Sources/ApolloCodegenLib/ApolloCodegenConfiguration.swift

```diff
@@ -210,22 +251,12 @@ public struct ApolloCodegenConfiguration {
   public let input: FileInput
   /// The paths and files output by code generation.
   public let output: FileOutput
-  /// Any non-default rules for pluralization or singularization you wish to include.
-  public let additionalInflectionRules: [InflectionRule]
-  /// Formatting of the GraphQL query string literal that is included in each
-  /// generated operation object.
-  public let queryStringLiteralFormat: QueryStringLiteralFormat
-  /// How to handle properties using a custom scalar from the schema.
-  public let customScalarFormat: CustomScalarFormat
```

Comment:
You might notice that this property is not present in `OutputOptions`. That is because the workflow for custom scalars in code generation 1.0 is different; by default custom scalars are always a `typealias` to `String` and the user is free to edit that generated file without being overwritten. So we no longer need an options for default or passthrough behaviour.

Reply:
I love this new behavior :) I hope users like it too haha.